client/sources/ok_test/models.py

```diff
@@ -100,9 +100,11 @@ def unlock(self, interact):
                                 for case in suite.cases])
         for suite_num, suite in enumerate(self.suites):
             for case_num, case in enumerate(suite.cases):
+                case_id = '{} > Suite {} > Case {}'.format(
```

Comment:
My requirement is that I need a given line of data to uniquely identify across the course what question it is. A hash of the question contents + the assignment ID is potentially insufficient for that, if the question content is the same as another within an assignment.

The students changing things is a problem admittedly. Would introducing a unique ID be difficult?